torch/_dynamo/variables/tensor.py

```diff
@@ -263,6 +263,9 @@ def call_method(
         args: "List[VariableTracker]",
         kwargs: "Dict[str, VariableTracker]",
     ) -> "VariableTracker":
+        if tx.strict_checks_enabled:
+            if name in ["stride"]:
+                unimplemented(f"Illegal method invocation {name} in strict mode")
```

Comment:
We should probably also disable all Tensor properties besides size, because those are unknown.

Off of the top of my head: storage_offset, requires_grad (the attribute), dtype, device, layout, all the `is_*` functions (e.g. is_cuda, is_quantized, ...).

Ideally we'd also have a check for `isinstance(x, <something that is not a torch.Tensor>)`, but that is also unlikely in practice.

Reply:
We can't get here without it being a tensor, but yes, agreed with the rest.